qcengine/compute.py

```diff
@@ -76,16 +76,29 @@ def compute(input_data: Union[Dict[str, Any], 'ResultInput'],
         local_options = {**local_options, **input_engine_options}
         config = get_config(local_options=local_options)
 
-        output_data = executor.compute(input_data, config)
+        for x in range(config.retries + 1):
+            try:
+                output_data = executor.compute(input_data, config)
+                break
+            except RandomError as e:
+
+                if x == config.retries:
+                    raise e
+                else:
+                    metadata["retries"] += 1
+            except:
+                raise
```

Comment:
is the last except/raise just to be explicit about flow control?

Reply:
This will raise a non-Random Exception. For example there is not a good reason to run an InputError multiple times.